client/pages/ListBikes.tsx

```diff
@@ -64,54 +67,62 @@ const ListBikes = ({route}) => {
                 backgroundColor='white'
                 barStyle='dark-content'
             />
-            <View style={styles.searchRow}>
-                <View style={styles.dropdownWrapper}>
-                    <Text style={styles.dropdownLabel}>Filter Search by Tags</Text>
-                    <DropDownPicker
-                        maxHeight={300}
-                        style={styles.dropdownInput}
-                        multiple={true}
-                        min={0}
-                        max={3}
-                        open={open}
-                        value={tags}
-                        items={items}
-                        setOpen={setOpen}
-                        setValue={setTags}
-                        setItems={setItems}
-                        mode='BADGE'
-                        onChangeValue={filterBikes}
-                        showBadgeDot={false}
-                    />
-                </View>
-                <TouchableOpacity
-                    // Had to apply type 'never' to string param for navigate.
-                    // Seems like a weird React/TypeScript issue and this is
-                    // a quick workaround as found here:
-                    // https://stackoverflow.com/questions/68667766/react-native-typescript-string-is-not-assignable-to-parameter-of-type-never
-                    onPress={() => {
-                        navigation.navigate(
-                            'Search' as never,
-                            { screen: 'Map' } as never
-                        )
-                    }
-                    }
-                >
-                    <Ionicons
-                        name='map'
-                        size={40}
-                        color='black'
-                    />
-                </TouchableOpacity>
-            </View>
-
-            <FlatList style={styles.bikesWrapper}
-                ListEmptyComponent={handleEmpty}
-                keyExtractor={item => item.bike_id}
-                data={selectedBikes}
-                extraData={selectedBikes}
-                renderItem={({ item }) => (<BikeItem bike={item} hasLink={true}></BikeItem>)}
-            />
+                {
+                    isLoading ?
+                        <View style={styles.spinnerContainer}>
```

Comment:
I know this is a super small amount of code, but maybe we could make a separate component out of it so we can use it in other screens for some stylistic consistency? I'd love to add it to the map screen.